### PR TITLE
qm-cloud-init : add page

### PR DESCRIPTION
**Pull request by AaryaZ**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Welcome to tldr and thanks for your contribution. In future, please read the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) before creating a PR. In this case, I will manually update your page and fix it.

**Comment by AaryaZ** [comment]
@kbdharun I will make the changes according to the style guide.

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
> @kbdharun can you let me know what changes should be done so that the pr will be merged. also can you add hacktoberfest label to the pr

This PR will get merged once another maintainer approves it. And this PR is valid for hacktoberfest without adding the label as we add hacktoberfest topic to our repositories by default. Your contribution once merged will get counted over there after a 7 day wait period.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**
